.gitignore

```diff
@@ -9,6 +9,7 @@ build/
 npm-debug.log
 node_modules/
 src/js/config/Config.dev.js
```

Comment:
We don't really need both of them do we?

Reply:
I thought it would be nicer for people migrating over (got a lot of bad vibes for not ignoring `jest/config.json` although we didn't need it anymore)